src/API/Reports/Orders/Controller.php

```diff
@@ -475,6 +475,16 @@ public function get_collection_params() {
 		return $params;
 	}
 
+	/**
+	 * Get customer name column export value.
+	 *
+	 * @param array $customer Customer from report row.
+	 * @return string
+	 */
+	protected function _get_customer_name( $customer ) {
```

Comment:
We're still supporting PHP 7.0.0, which doesn't support specifying the visibility of a function. You also don't need to indicate visibility using a prefixed underscore - so `function get_customer_name( $customer )` should be fine here.

Reply:
Sorry! I misread the PHP documentation, the visibility of a function is fine to use. If you drop that underscore it should be fine to merge.